packages/angular/src/builders/webpack-browser/webpack-browser.impl.ts

```diff
@@ -31,7 +51,8 @@ function buildApp(
       return buildAppWithCustomWebpackConfiguration(
         delegateOptions,
         context,
-        pathToWebpackConfig
+        pathToWebpackConfig,
+        pathToIndexFileTransformer ?? undefined
```

Comment:
nit: the `?? undefined` is not needed, you're handling any falsy value in the usage.

Reply:
ah good point